config/localization/en/app.json

```diff
@@ -1519,6 +1519,31 @@
       "YOUR_LEAGUE": "Your League",
       "YOURE_IN": "You're in {LEAGUE}"
     },
+    "VAULTS": {
+      "VAULT_DESCRIPTION": "Vaults provide automated liquidity on all dYdX markets. Vaults aim for a market-neutral position by quoting both sides of the book. P&L will vary based on market conditions and there's a risk of losing some or all of the USDC deposited.",
+      "LP_VAULT": "LP Vault",
+      "VAULT": "Vault",
+      "ASSET_VAULT": "{ASSET} Vault",
+      "VAULTS": "Vaults",
+      "VAULT_DETAILS": "Vault Details",
+      "VAULT_FAQS": "Vault FAQs",
+      "VAULT_BALANCE": "Vault Balance",
+      "VAULT_POSITION": "Vault Position",
+      "VAULT_LEVERAGE": "Vault Leverage",
+      "VAULT_ALL_TIME_PNL": "All-time Vault P&L",
+      "VAULT_THIRTY_DAY_APR": "30d APR",
```

Comment:
v slight nit, I wonder if it's better to do something like `"TIME_APR": "{TIME_DIFF} APR"` and then plug in 30d with our truncation number format thing in v4web to make it more reusable / adjustable.

Reply:
In this case we only ever render 30d so should be okay, but if I end up needing other days I'll do the flexible version.